style.css

```
@@ -0,0 +1,114 @@
+body{
+  width: 100%;
+  max-width: 2180px;
+  margin: 0 auto;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size:  18px;
+  font-weight: 400;
+  line-height: 150%;
+  text-decoration: none;
+  background-color: #274001;
+  color: #BF9B6F;
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  }
```

Впринципе все хорошо только я бы добавил еще padding  что бы были отступы в о всем теле 
```suggestion
body{
  width: 100%;
  max-width: 2180px;
  margin: 0 auto;
  font-family: Verdana, Geneva, Tahoma, sans-serif;
  font-size:  18px;
  font-weight: 400;
  line-height: 150%;
  text-decoration: none;
  background-color: #274001;
  color: #BF9B6F;
  display: flex;
  flex-direction: column;
  padding: 20px;
  }
```